client/src/components/Places/index.js

```diff
@@ -10,19 +10,28 @@ class Places extends React.Component {
   }
 
   componentDidMount() {
-    this.setState({ initialList: this.props.listOfItems, filteredItems: this.props.listOfItems })
+    const itemList = sessionStorage.getItem('places')
+    const dataParsed = JSON.parse(itemList)
+    console.log('list', dataParsed)
+
+    this.setState({
+      initialList: dataParsed,
+      filteredItems: dataParsed
```

Comment:
```suggestion
      filteredItems: parsedData
```

Reply:
both filteredItems and initialList have the same value. We likely can scrap one of them. (let me know if you can't figure it out)